qcelemental/tests/test_molecule.py

```diff
@@ -240,10 +244,10 @@ def test_water_orient():
 
 
 def test_molecule_errors_extra():
-    data = water_dimer_minima.dict()
+    data = water_dimer_minima.dict(skip_defaults=True)
     data["whatever"] = 5
     with pytest.raises(Exception):
-        Molecule(**data)
+        Molecule(**data, validate=False)
```

Comment:
Why do we validate=False here if we expect this to raise?

Reply:
Because validate will clean the input arrays.